lab_1_classify_by_unigrams/main.py

```diff
@@ -50,6 +77,20 @@ def create_language_profile(language: str, text: str) -> dict[str, str | dict[st
     In case of corrupt input arguments, None is returned
     """
 
+    # checking input
+    if not isinstance(language, str) or not isinstance(text, str):
+        return None
+
+    frequencies = calculate_frequencies(tokenize(text))
+    if (not isinstance(frequencies, dict)
+            or not all(isinstance(char, str) for char in frequencies)
+            or not all(isinstance(value, float) for value in frequencies.values())):
+        return None
+
+    if frequencies is not None:
```

Comment:
may be you can swap the conditions, make here early stop

Reply:
done